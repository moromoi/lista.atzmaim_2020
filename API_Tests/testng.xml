<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd"><suite name="All Test Suite"><test verbose="2" preserve-order="true" name="D:/Devel_lista2020/lista.atzmaim_2020/API_AccountCreation_DDT_Tests/src/test/resources/input.xlsx"><classes/></test></suite>