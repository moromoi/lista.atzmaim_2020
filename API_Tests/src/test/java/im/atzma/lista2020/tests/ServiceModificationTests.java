package im.atzma.lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.Assert;
import org.testng.annotations.Test;

public class ServiceModificationTests extends TestBase {

    @Test
    public void modificateService() {
        int category_id = app.serviceCreationHelper().createCategory();
        int service_id = app.serviceCreationHelper().createService(category_id);
        app.serviceCreationHelper().modifyService(service_id, category_id);

        Approvals.verify(app.serviceCreationHelper().getServiceList());

    }

}
