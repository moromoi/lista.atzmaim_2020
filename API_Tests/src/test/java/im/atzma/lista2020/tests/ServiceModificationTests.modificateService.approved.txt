Server=nginx
#####
Content-Type=application/json; charset=utf-8
Transfer-Encoding=chunked
Connection=keep-alive
Vary=Accept-Encoding
Set-#####
X-Request-#####
Strict-Transport-Security=max-age=604800
X-Content-Type-Options=nosniff
X-Robots-Tag=noindex
Content-Security-Policy=upgrade-insecure-requests
X-Powered-By=php/4.3.10-16
Content-Encoding=gzip

[
    {
        "#####
        "name": "TestService_modified",
        "price": "40",
        "duration": 70,
        "color": "#50e3c1",
        "category": {
            "name": "TestCategory",
            "#####
        }
    }
]